test/Microsoft.Health.Fhir.Tests.Integration/Features/Operations/Export/CreateExportRequestHandlerTests.cs

```diff
@@ -0,0 +1,151 @@
+using Microsoft.Health.Fhir.Tests.Integration.Persistence;
+using Xunit;
+
+namespace Microsoft.Health.Fhir.Tests.Integration.Features.Operations.Export
+{
+    [Collection(FhirOperationConstants.FhirOperationTests)]
+    [FhirStorageTestsFixtureArgumentSets(DataStore.CosmosDb)]
+    public class CreateExportRequestHandlerTests : IClassFixture<FhirStorageTestsFixture>, IAsyncLifetime
+    {
+        private static readonly Uri RequestUrl = new Uri("https://localhost/$export/");
+        private const string DestinationType = "destinationType";
+        private const string ConnectionString = "destinationConnection";
+
+        private readonly MockClaimsExtractor _claimsExtractor = new MockClaimsExtractor();
+        private readonly ISecretStore _secretStore = new InMemorySecretStore();
+        private readonly IFhirOperationDataStore _fhirOperationDataStore;
+        private readonly IFhirStorageTestHelper _fhirStorageTestHelper;
+
+        private readonly CreateExportRequestHandler _createExportRequestHandler;
+
+        private readonly CancellationToken _cancellationToken = new CancellationTokenSource().Token;
+
+        public CreateExportRequestHandlerTests(FhirStorageTestsFixture fixture)
+        {
+            _fhirOperationDataStore = fixture.OperationDataStore;
+            _fhirStorageTestHelper = fixture.TestHelper;
+
+            _createExportRequestHandler = new CreateExportRequestHandler(_claimsExtractor, _fhirOperationDataStore, _secretStore);
+        }
+
+        public Task InitializeAsync()
+        {
+            return _fhirStorageTestHelper.DeleteAllExportJobRecordsAsync();
+        }
+
+        public Task DisposeAsync()
+        {
+            return Task.CompletedTask;
+        }
+
+        [Fact]
+        public async Task GivenThereIsNoMatchingJob_WhenCreatingAnExportJob_ThenNewJobShouldBeCreated()
+        {
+            var request = new CreateExportRequest(RequestUrl, DestinationType, ConnectionString);
+
+            CreateExportResponse response = await _createExportRequestHandler.Handle(request, _cancellationToken);
+
+            Assert.NotNull(response);
+            Assert.NotEmpty(response.JobId);
+
+            SecretWrapper secret = await _secretStore.GetSecretAsync($"Export-Destination-{response.JobId}");
+
+            Assert.NotNull(secret);
+        }
+
+        [Fact]
+        public async Task GivenThereIsAMatchingJob_WhenCreatingAnExportJob_ThenNewJobShouldBeCreated()
+        {
+            var request = new CreateExportRequest(RequestUrl, DestinationType, ConnectionString);
+
+            CreateExportResponse response = await _createExportRequestHandler.Handle(request, _cancellationToken);
+
+            var newRequest = new CreateExportRequest(RequestUrl, DestinationType, ConnectionString);
+
+            CreateExportResponse newResponse = await _createExportRequestHandler.Handle(request, _cancellationToken);
+
+            Assert.NotNull(newResponse);
```

Comment:
nit: should we also assert the `response` isn't null?

Reply:
It's already verified by the test above.